package.json

```diff
@@ -4,50 +4,52 @@
   "description": "IPFS Native Application",
   "main": "src/index.js",
   "dependencies": {
-    "css-box-sizing-border-box": "^0.1.0",
-    "ipfs-api": "^9.0.0",
+    "ipfs-api": "^17.1.2",
     "ipfs-geoip": "^2.1.0",
     "ipfs-logo": "github:ipfs/logo",
-    "ipfsd-ctl": "^0.16.0",
+    "ipfsd-ctl": "^0.25.1",
     "menubar": "^5.0.0",
     "minimist": "^1.2.0",
-    "multiaddr": "^2.0.2",
-    "node-notifier": "^4.3.1",
-    "normalize.css": "^5.0.0",
-    "radium": "^0.18.0",
-    "react": "^15.1.0",
+    "multiaddr": "^3.0.1",
+    "node-notifier": "^5.1.2",
+    "normalize.css": "^7.0.0",
+    "prop-types": "^15.6.0",
+    "radium": "^0.19.6",
+    "react": "^16.1.1",
     "react-addons-css-transition-group": "^15.1.0",
     "react-dnd": "^2.1.2",
     "react-dnd-html5-backend": "^2.1.2",
-    "react-dom": "^15.1.0",
-    "react-widgets": "^3.2.0",
+    "react-dom": "^16.1.1",
+    "react-widgets": "^4.1.1",
     "winston": "^2.1.0"
   },
   "devDependencies": {
     "babel-core": "^6.3.21",
```

Comment:
While you are at it, can you add `aegir` as a devDep and use its `aegir lint` to make sure the linting rules are the same as the remaining modules?

Reply:
Should I add it to webui too?

Reply:
Done done 😄